### ISSUE TITLE
Couldn't find a configuration settings named "registry"

### ISSUE THREAD

**Issue by PerchunPak** [withastro/astro#7404]
### What version of `astro` are you using?

2.6.3

### Are you using an SSR adapter? If so, which one?

Doesn't matter

### What package manager are you using?

yarn

### What operating system are you using?

Manjaro Linux

### What browser are you using?

Doesn't matter

### Describe the Bug

When running `yarn astro add svelte`, it executes `yarn config get registry` which fails.

```
 yarn astro add svelte
✖ Command failed with exit code 1: yarn config get registry
Usage Error: Couldn't find a configuration settings named "registry"

$ yarn config get [--json] [--no-redacted] <name>
```

### Link to Minimal Reproducible Example

can't find a way to get yarn v3 in stackblitz

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by ematipico** [comment]
Could you create a minimal repository where we can test the issue?

**Comment by bigmistqke** [comment]
same error here with `npx astro add react`

**Comment by PerchunPak** [comment]
Send the full output of command please, this shouldn't be possible with `npm`.

**Comment by bigmistqke** [comment]
```
npx astro add react
✖ Command failed with exit code 1: yarn config get registry
Usage Error: Couldn't find a configuration settings named "registry"

$ yarn config get [--json] [--no-redacted] <name>
```

I don't know why it's trying to use yarn, maybe unrelated issue.

**Comment by PerchunPak** [comment]
Do you have `yarn.lock`?

**Comment by bigmistqke** [comment]
No. I used yarn earlier, but there is no more yarn-file in my repo. 
I am having a monorepo with npm workspaces.

**Comment by natemoo-re** [comment]
I believe this has been fixed by https://github.com/withastro/astro/pull/7527, which should be released soon.
